lib/template.html

```diff
@@ -51,7 +51,11 @@
 
 {% if site.twitter %}
   {% if seo_tag.image %}
-    <meta name="twitter:card" content="summary_large_image" />
+    {% if page.twitter_image_small or post.twitter_image_small %}
+      <meta name="twitter:card" content="summary" />
+    {% else %}
+      <meta name="twitter:card" content="summary_large_image" />
+    {% endif %}
```

Comment:
Per the discussion over in https://github.com/jekyll/jekyll-seo-tag/pull/222#discussion_r131346046, would it make more sense to implement this as something like:

```liquid
<meta name="twitter:card" content="{{ page.twitter.card | default: site.twitter.card | default: "summary_large_image" }}" />
```

That way if Twitter adds a "medium" card next year, we're still covered.

Reply:
Having been 6 months on this...is the only way to get this in to force the user to put the exact card type into their front matter?

To repeat why I went with this approach, Twitter has 2 card types that would work with the rest of the meta this plugin adds.
By using a Boolean option, the user doesn't need to know the exact twitter card names (which is one plus for this whole plugin, the user doesn't need to know all the meta tags)

I would appreciate a definite change to get in, or a merge as is so this can move forward please.

Reply:
The boolean seems clunky. Ie: Why `twitter_image_small: true` instead of `twitter_image_large: false`?

Let's go with `twitter.card: summary_large_image` :+1: